docs/pages/versions/unversioned/sdk/font.mdx

```diff
@@ -19,6 +24,46 @@ import { SnackInline } from '~/ui/components/Snippet';
 
 <APIInstallSection />
 
+## Configuration in app.json/app.config.js
```

Comment:
```suggestion
## Configuration in app config
```

Reply:
i copied this section header from the expo-camera docs, i believe we use this same heading in each docs page though

https://github.com/expo/expo/blob/main/docs/pages/versions/unversioned/sdk/camera.mdx#L30

Reply:
Interesting, in some places we are using app config: https://docs.expo.dev/versions/latest/sdk/apple-authentication/#configuration-in-app-config

Looks like I need to update cause, in some places, we are using app config. Let's ignore this suggestion for now. I'll update it in other places and this one altogether.